README.md

```diff
@@ -25,7 +25,10 @@ Opinionated conventions and best practices:
 * [7val/rancher-deployment]: Run a deployment to [Rancher][1] via generated
   `docker-compose.yml` and `rancher-compose.yml` files.
 * [7val/sloppy]: Provides the Slopp.io CLI in a container
-* [7val/sloppy-deployment]: Run a Sloppi.io deployment
+* [7val/sloppy-deployment]: Run a Sloppy.io deployment with the help of template
+  configuration and environment variables.
+* [7val/sloppy-rollout]: Enhances `7val/sloppy-deployment`. Plain JSON/YAML
```

Comment:
You need to add the definition of the reference `[7val/sloppy-rollout]` at the end of the file. Otherwise it is not a clickable link.

Reply:
fixed